src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const addFunction = (nums) => {
```

Comment:
you get a one number here, not array
```suggestion
  const addFunction = (number) => {
```